scripts/import_currencies.py

```
@@ -0,0 +1,75 @@
+    "BAM": "KM", "BBD": "$", "BDT": "৳", "BGN": "лв", "BHD": ".د.ب",
+    "BIF": "Fr", "BMD": "$", "BND": "$", "BOB": "Bs.", "BRL": "R$",
+    "BSD": "$", "BTN": "Nu", "BWP": "P", "BYN": "Br", "BZD": "$",
+    "CAD": "$", "CDF": "Fr", "CHF": "Fr", "CLP": "$", "CNY": "¥",
+    "COP": "$", "CRC": "₡", "CUP": "$", "CVE": "$", "CZK": "Kč",
+    "DJF": "Fr", "DKK": "kr", "DOP": "$", "DZD": "د.ج", "EGP": "£",
+    "ERN": "Nfk", "ETB": "Br", "EUR": "€", "FJD": "$", "FKP": "£",
+    "GBP": "£", "GEL": "₾", "GHS": "₵", "GIP": "£", "GMD": "D",
+    "GNF": "Fr", "GTQ": "Q", "GYD": "$", "HKD": "$", "HNL": "L",
+    "HRK": "kn", "HTG": "G", "HUF": "Ft", "IDR": "Rp", "ILS": "₪",
+    "INR": "₹", "IQD": "ع.د", "IRR": "﷼", "ISK": "kr", "JMD": "$",
+    "JOD": "د.ا", "JPY": "¥", "KES": "Ksh", "KGS": "с", "KHR": "៛",
```

Ruff (0.15.12)

[warning] String contains ambiguous `ا` (ARABIC LETTER ALEF). Did you mean `l` (LATIN SMALL LETTER L)?

(RUF001)